cirq/experiments/qubit_characterizations_test.py

```diff
@@ -72,6 +81,11 @@ def check_distinct(unitaries):
         assert any(
             cirq.allclose_up_to_global_phase(Uxy, Uxz) for Uxz in xz_unitaries)
 
+    # Check that each unitary fixes the Pauli group.
+    for u in xy_unitaries:
+        for p in PAULIS:
+            assert is_pauli(u @ p @ u.conj().T), f'{u}'
```

Comment:
nit: instead of `f'{u}'` can just do `str(u)`

Reply:
Done.